src/sonic-bgpcfgd/tests/test_advertise_rt.py

```diff
@@ -62,7 +62,7 @@ def test_set_del():
     set_del_test(
         mgr,
         "SET",
-        ("fc00:10::/64", {}),
```

Comment:
Can we retain this case to cover empty data?

Reply:
@prsunny  I think the old unit test is inaccurate, from existing code of VNetRouteOrch, it send out {key:{"":""}} to the bgpcfgd for empty case as below. And also by manually testing,  can't insert a key with a "empty" to the redis.
void VNetRouteOrch::addRouteAdvertisement(IpPrefix& ipPrefix)
{
    const string key = ipPrefix.to_string();
    vector<FieldValueTuple> fvs;
    fvs.push_back(FieldValueTuple("", ""));
    state_vnet_rt_adv_table_->set(key, fvs);
}